testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/groups/GroupTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * JBoss, Home of Professional Open Source
+ *
+ * Copyright 2013 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.testsuite.console.groups;
+
+import org.jboss.arquillian.graphene.page.Page;
+import org.junit.Before;
+import org.junit.Test;
+import org.keycloak.representations.idm.GroupRepresentation;
+import org.keycloak.testsuite.console.page.groups.CreateGroup;
+
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author clementcur
+ */
+public class GroupTest extends AbstractGroupTest {
+
+    @Page
+    private CreateGroup createGroupPage;
+
+    @Before
+    public void beforeCreateGroupTest() {
+        groupsPage.navigateTo();
+    }
+
+    @Test
+    public void createAGroup() {
```

Comment:
This test seem to be failing. Needs to be passing with FF and Chrome.

Reply:
fixed in latest commit